Profiles - ZIB 2017/zib-AdministrationAgreement.xml

```diff
@@ -277,7 +277,7 @@
       <path value="MedicationDispense.category.coding" />
       <sliceName value="administrationAgreementCode" />
       <short value="AdministrationAgreementCode" />
-      <definition value="Category snomed code used for indexing/retrieval of the ZIB AdministrationAgremeent. The snomed code 422037009 for this resource is stated in the Medication Process v09." />
+      <definition value="Category SNOMED code used for indexing/retrieval of the HCIM AdministrationAgreement. The SNOMED code 422037009 for this resource is stated in the Medication Process v09." />

```

Comment:
I think it's better to handle this change using a proper change request.

Reply:
See https://github.com/Nictiz/Nictiz-STU3-Zib2017/pull/307